zdm/MCMC.py

```diff
@@ -14,7 +14,6 @@
 
```

Comment:
I've made this explicit in the comments now. MCMC uses updates while MCMC2 re-initialises the grid on every iteration (which is generally faster than trying to copy a huge grid in memory). The 'wrap' versions just handle all the command line prompts so that I can test the MCMC2.py functions (explicitly call them in other files - this doesn't work if you have argparser things).